pkg/types/aws_resource_manager.go

```diff
@@ -89,4 +89,7 @@ type AWSResourceManagerFactory interface {
 	) (AWSResourceManager, error)
 	// IsAdoptable returns true if the resource is able to be adopted
 	IsAdoptable() bool
+	// GetRequeueOnSuccessSeconds returns true if the resource should be requeued after specified seconds
+	// Default is false which means resource will not be requeued after success.
+	GetRequeueOnSuccessSeconds() (int, bool)
```

Comment:
@mbaijal how about we simplify the method signature to just returning an `int` and if the default (`0`) is returned, then no requeue happens.

Also, let's call this just `RequeueOnSuccessSeconds()` (without the `Get` prefix) to align with the other method names :)

Reply:
done.